### PR TITLE
fix: only allow EIP155 transaction

### PR DESCRIPTION
**Pull request by RetricSu**

### CODE REVIEW

**Review by classicalliu — APPROVED** [review]

File: docs/compatibility.md [hunk 1]
```diff
@@ -17,6 +17,19 @@ Since there is no native token in Godwoken Polyjuice(while Ethereum has ETH as i
 
 - Use the `transfer function` in [CKB_ERC20_Proxy](https://github.com/nervosnetwork/godwoken-polyjuice/blob/3f1ad5b/solidity/erc20/README.md) contract [combined](https://github.com/nervosnetwork/godwoken-polyjuice/blob/3f1ad5b322/solidity/erc20/SudtERC20Proxy_UserDefinedDecimals.sol#L154) with sUDT_ID = 1 (CKB a.k.a. pETH).
 
+### Signing Transaction Only Support EIP155
+
+[EIP155](https://eips.ethereum.org/EIPS/eip-155) add the chainId for simple replay attack protection. currently we only support EIP155 signing scheme.
```
Comment (Flouse) [inline docs/compatibility.md:22]:
```suggestion
[EIP155](https://eips.ethereum.org/EIPS/eip-155) add the chainId for simple replay attack protection. Currently we only support EIP155 signing scheme.
```

File: docs/compatibility.md [hunk 2]
```diff
@@ -17,6 +17,19 @@ Since there is no native token in Godwoken Polyjuice(while Ethereum has ETH as i
...
+
+### Result
```
Comment (Flouse) [inline docs/compatibility.md:24]:
```suggestion
### Known issue #238
```

File: docs/compatibility.md [hunk 3]
```diff
@@ -17,6 +17,19 @@ Since there is no native token in Godwoken Polyjuice(while Ethereum has ETH as i
...
+
+use outdated ethereum toolchain like `truffle-hdwallet-provider` to send transaction will result failed. 
+
+issue: #238
```
Comment (Flouse) [inline docs/compatibility.md:28]:
```suggestion
```

File: docs/compatibility.md [hunk 4]
```diff
@@ -17,6 +17,19 @@ Since there is no native token in Godwoken Polyjuice(while Ethereum has ETH as i
 
 - Use the `transfer function` in [CKB_ERC20_Proxy](https://github.com/nervosnetwork/godwoken-polyjuice/blob/3f1ad5b/solidity/erc20/README.md) contract [combined](https://github.com/nervosnetwork/godwoken-polyjuice/blob/3f1ad5b322/solidity/erc20/SudtERC20Proxy_UserDefinedDecimals.sol#L154) with sUDT_ID = 1 (CKB a.k.a. pETH).
 
+### Signing Transaction Only Support EIP155
+
+[EIP155](https://eips.ethereum.org/EIPS/eip-155) add the chainId for simple replay attack protection. currently we only support EIP155 signing scheme.
+
+### Result
+
+use outdated ethereum toolchain like `truffle-hdwallet-provider` to send transaction will result failed.
```
Comment (Flouse) [inline docs/compatibility.md:26]:
```suggestion
Using outdated Ethereum toolchain like `truffle-hdwallet-provider` to send transaction will result in failure. 
```

**Review by Flouse — CHANGES_REQUESTED** [review]

File: docs/compatibility.md (hunk 2 above)
Comment (RetricSu) [reply docs/compatibility.md:24]: issue 234 can not be click if change to this?

**Review by Flouse — APPROVED** [review]

**Pull request merged**